src/main.rs

```diff
@@ -1,4 +1,5 @@
 use clap::{Parser, Subcommand};
+use noir_libs::ops::{add, remove};
 
 /// A CLI package manager for Noir
```

Comment:
A CLI package manager for Noir | noir-libs.org

Reply:
Fixed